### PR TITLE
Responsive implementation

### PR DESCRIPTION
**Pull request by vz3**
High level changes
--
- New 24 column grid: 24 col $large, 12 col $medium, 4 col $small, 4 col $x-small. (The small grid may move up to 8 column in the future).
- New mixin for center-aligned narrow columns. Usage: ```@include inner($columns);``` Elements using this mixin become full-width on small view.
- New "container" class borrowed from @noahmanger/[MyRA](https://github.com/18f/myra). Apply this class as a div around all non-full bleed elements to turn it into an outer-container with the appropriate styling.
- Incorporates [normalize.css 3.0.2](http://necolas.github.io/normalize.css/)

Existing issues
--
- Nav formatting is off (menu vertical alignment)
- Mobile nav view isn't implemented
- 18F footer breaks on nav
- Long subdetails on agency landing page breaks layout
- Banner should be honed in

Next steps
--
- Address existing issues
- Incorporate bitters for styling.

---

#### Original PR notes

Opening this request, but needs to be tested with new #327 code. I'm unable to do that at this time without a local instance of that app…

This is all changes to SCSS based upon the HTML of the current foia.18f.us

The biggest change is that it adds a mobile, (unused) tablet, and desktop view. The desktop view is using a 12 column grid, while previously we were using 7 columns and shifting by 1.5. Mobile moves down to a 4 column grid. This had no ramifications with the current codebase, but conflicts may be present with #327

This also incorporates [normalize.css 3.0.2](http://necolas.github.io/normalize.css/)

### CODE REVIEW

**Comment by rjmajma** [comment]
Pinging to see if @vz3 has made any headway on this. What's the status, pal?

**Comment by vz3** [comment]
Last task here is fixing the agency landing page view and styling the US Gov banner, rest of the site should be working here.
screen shot 2015-01-02 at 5 36 01 pm

**Comment by vz3** [comment]
There's some minor visual glitches left (vertical alignment of nav needs to be honed in, there's still a problem with subdetails dropping down to a new line on mobile), but this is probably in a place that makes sense to review.

**Comment by khandelwal** [comment]
@vz3 This pull request is still assigned to you (which indicates that it's not ready for review). However, I think in the comment above you're suggested it be reviewed? Did you mean to unassign it? Or are you going to keep working on the glitches?

**Comment by vz3** [comment]
@khandelwal I think this should be merged upon review. Outstanding issues are documented and relatively minor design issues.

Because this touches so much on the front end (e.g. a new 24-column grid), any separate template work based on the current master branch is at risk until the merge happens.

File: README.md [hunk 1]
```diff
@@ -125,6 +125,8 @@ git clone git@github.com:18F/foia.git
 Then run the data loading script:
 
 ```bash
+cd foia-hub
```
Comment (khandelwal) [inline README.md]: On  adding this, do you mean to remove the two similar lines below?

**Comment by khandelwal** [comment]
Someone who is not me needs to review this. @konklone?

**Comment by vz3** [comment]
Ha yes any edits to the readme can be dropped

**Comment by konklone** [comment]
OK, I dropped the edits to the README.

@vz3, this looks good. There's kind of an [awkward region on the landing page](https://cloud.githubusercontent.com/assets/4592/5675238/3f2bf46e-9787-11e4-9cee-ae86a5a95b4b.png), but it's wider than typical mobile viewports and an unlikely width for anyone to be at.

**Comment by konklone** [comment]
I'm merging, since my changes to the README removed it from the PR entirely.

**Pull request merged**